code/_expts/slotnick_etal_2001/analyze/plots/de_PlotGroupBasicsSlot.m

```diff
@@ -28,7 +28,7 @@
             ax(tai) = subplot(1, num_ttypes, tai);
             de_CreateSlotnickFigure1([left_mean right_mean], ...
                                      [left_stderr right_stderr], ...
-                                     taskTitle, stimSet, runs, ax(tai));
+                                     taskTitle, stimSet, runs,runs, ax(tai));
```

Comment:
This is only correct if there have been no model rejections. These should either use the # of models, or just pass a single parameter.